biz.aQute.bndlib/src/aQute/bnd/build/Workspace.java

```diff
@@ -610,6 +610,7 @@ protected void setTypeSpecificPlugins(Set<Object> list) {
 
 			try {
 				HttpClient client = new HttpClient();
+				client.setOffline(offline);
```

Comment:
Can you use `getOffline()` here? I prefer using methods instead of direct access to fields.

```java
client.setOffline(getOffline());
```